### PR TITLE
PISTON-616:Added did classifier to cdr for outgoing offnet calls

### PR DESCRIPTION
**Pull request by bradfordben**

### CODE REVIEW

File: applications/stepswitch/src/stepswitch_resources.erl [hunk 1]
```diff
@@ -536,14 +536,15 @@ maybe_resource_to_endpoints(#resrc{id=Id
...
-            lager:debug("building resource ~s endpoints (classifier ~s)", [Id, _MaybeClassifier]),
+            MaybeClassifier = case kz_term:is_ne_binary(Classifier) of
+                                  'true' -> Classifier;
+                                  'false' -> <<"no_classifier">>
+                              end,
+            lager:debug("building resource ~s endpoints (classifier ~s)", [Id, MaybeClassifier]),
             CCVUpdates = [{<<"Global-Resource">>, kz_term:to_binary(Global)}
                          ,{<<"Resource-ID">>, Id}
                          ,{<<"E164-Destination">>, Number}
+                         ,{<<"DID-Classifier">>, get_did_classifier(Number, MaybeClassifier)}
```
[the remark this answers is not in the document]
Comment (bradfordben) [reply applications/stepswitch/src/stepswitch_resources.erl]: Hi, Thanks for your reply and explanation, i was not sure if there was a difference between the resource classification and the knm classification. The intention is just to capture and add the DID classifier so you are right it should always be the value of knm_converters:classify(Number). I will update the PR

[the remark this answers is not in the document]
Comment (bradfordben) [reply applications/stepswitch/src/stepswitch_resources.erl]: @k-anderson i was wondering if you has a chance to review this.

**Comment by bradfordben** [comment]
@k-anderson i was wondering if you has a chance to review this. Thanks, Ben

**Comment by jamesaimonetti** [comment]
Merged!

**Comment by bradfordben** [comment]
Thanks 👍

**Pull request closed without merging**